### ISSUE TITLE
"Blank" attention plots

### ISSUE THREAD

**Issue by gabriel-souza-omni** [fatchord/WaveRNN#154]
Hello. I am trying to train a portuguese speaking model, and while the audios generated by the wavenet during its training sound pretty good, when trying to generalize the audio sounds garbled. The only apparent issue is the attention plots shown below. Do you know what could cause this?

![3gufu6](https://user-images.githubusercontent.com/53836379/69158853-ccdb9880-0ac5-11ea-9e18-058e4b7d6adb.gif)

**Comment by mindmapper15** [comment]
I think you have the same problem with this.
https://github.com/fatchord/WaveRNN/issues/77

Also, there is another attention class you can use called Attention() in models/tacotron.py
I think it is Bahdanau Attention. You might have to change LSA to this one in Decoder class.

I'm currently training with Bahdanau Attention with my korean voice dataset and this is my attention result from 0 to 55k steps.

![ezgif-4-bcf665411da9](https://user-images.githubusercontent.com/40658807/69202213-46d25700-0b84-11ea-8a99-d52c3c93d002.gif)

**Comment by cschaefer26** [comment]
Hi, I also had trouble to build attention for some custom datasets. What helped best so far was:

1. In the LSA module change the sigmoid activation to softmax (scores = F.softmax(u, dim=1). After attention is build this could be reverted without losing attention.
2. Start the schedule with a larger reduction factor (e.g. 12 instead of 7)

**Comment by gabriel-souza-omni** [comment]
Thanks guys. These suggestions worked really well, and I managed to get the attention working.

**Comment by mindmapper15** [comment]
One more thing.
It will be good to trimming which removes the front and rear silence in your audio dataset.
This will help attention to find proper alignment between text and audio more easily.
